stream-chat-android-client/src/main/java/io/getstream/chat/android/client/ChatClient.kt

```diff
@@ -202,13 +202,18 @@ public class ChatClient internal constructor(
         return api.getGuestUser(userId, userName)
     }
 
+    @CheckResult
     public fun sendFile(
         channelType: String,
         channelId: String,
         file: File,
-        callback: ProgressCallback,
-    ) {
-        api.sendFile(channelType, channelId, file, callback)
+        callback: ProgressCallback? = null,
+    ): Call<String> {
+        return if (callback != null) {
+            api.sendFile(channelType, channelId, file, callback)
+        } else {
+            api.sendFile(channelType, channelId, file)
+        }
```

Comment:
This seems unnecessary now that `ChatApi` can handle nullable values for `callback` too